src/os/ogc/ogcservice.js

```diff
@@ -0,0 +1,397 @@
+goog.module('os.ogc.OGCService');
+goog.module.declareLegacyNamespace();
+
+const Promise = goog.require('goog.Promise');
```

Comment:
This shadows the browser's native `Promise` API. We don't actually use it anywhere, but I think we're gonna prefer using `GoogPromise` for this in the future.